README.md

```diff
@@ -2,12 +2,12 @@
 
 All TDR documentation is available [here](https://github.com/nationalarchives/tdr-dev-documentation)
 
-This project can be used to spin up a jenkins server using ECS. The ECS cluster is created using terraform and the jenkins configuration uses the [JCasC](https://jenkins.io/projects/jcasc/) plugin and the jenkins.yml file sets up the jenkins configuration.
+This project can be used to spin up a jenkins server using ECS. The ECS cluster is created using terraform and the jenkins configuration uses the [JCasC](https://jenkins.io/projects/jcasc/) plugin. We use this project to create two Jenkins instances. Integration Jenkins uses the jenkins.yml file for configuration and jenkins-prod.yml for the production Jenkins.
```

Comment:
Maybe add a paragraph and use a list:

> ... plugin.
> 
> We use this project to create two Jenkins instances. For configuration:
> * Integration Jenkins uses the jenkins.yml file; and
> * Production Jenkins uses the jenkins-prod.yml

Reply:
Yeah, that looks better. I've changed it.